ONSdigital/census-rh-ui
# CR 701 Change of Region notification

# Motivation and Context
Add a new notification page informing the user that the url/design has been switched when the user enters a UAC with a region value that does not match the current 'display region' ie a NI UAC on the English url

# What has changed
Add new page, and logic to check if it should be displayed depending on current display_region and region value from the case for the UAC entered.

# How to test?
Unit tests are included.
To test manually, enter a ni UAC on the /en/start/ url - page should be displayed.
Requires an update to the RH Cucumber tests to deploy, which is also in review

# Links
https://collaborate2.ons.gov.uk/jira/browse/CR-701
https://github.com/ONSdigital/census-rh-cucumber/pull/54
